content/php/concepts/math-functions/terms/sin/sinh.md

```diff
@@ -0,0 +1,62 @@
+---
+Title: 'sinh()'
+Description: 'Returns the hyperbolic sine of a number.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Development'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Hyperbolic Functions'
```

Comment:
Hyperbolic Functions is not a current tag on our list. Please delete the tag, or add it to the tags.md document.